dbms/programs/server/ReplicasStatusHandler.cpp

```diff
@@ -40,6 +40,10 @@ void ReplicasStatusHandler::handleRequest(Poco::Net::HTTPServerRequest & request
         /// Iterate through all the replicated tables.
         for (const auto & db : databases)
         {
+            /// Lazy database can not contain replicated tables
+            if (db.second->getEngineName() == "Lazy")
```

Comment:
I don't like this conditions, but it's hard to do without them.
It can be fixed by adding `supportReplicatedTables`, `supportMergeTreeTables`, `supportAlters`... to IStorage.

Reply:
Ok for now.

Reply:
Maybe it can contain replicated tables but we will not include them in status?